otherlibs/stdune/src/path.ml

```diff
@@ -1265,6 +1265,18 @@ let chmod t ~mode = Unix.chmod (to_string t) mode
 let follow_symlink path =
   Fpath.follow_symlink (to_string path) |> Result.map ~f:of_string
 
+let drop_prefix path ~prefix =
+  String.drop_prefix (to_string path) ~prefix:(to_string prefix)
+  |> Option.map ~f:(fun p ->
+         String.drop_prefix_if_exists ~prefix:"/" p |> Local.of_string)
```

Comment:
I'm not certain, but would it be better to use the path separator here? I'm wondering what happens with this code on windows.

Reply:
No, the current code is correct. We can use `/` on windows.